user-manual/transfer/import-metadata.rst

```diff
@@ -21,6 +20,7 @@ into the METS file. Metadata in the METS file is searchable in the
     * :ref:`Simple objects <simple-objects>`
     * :ref:`Compound objects <compound-objects>`
 * :ref:`Importing rights metadata with rights.csv <rights.csv>`
+* :ref:`Importing structural metadata with mets_structmap.xml <structmap.xml>`
```

Comment:
Just FYI, you've added a link here (`<structmap.xml>`) but haven't created the anchor on the page - I've added it as a suggestion farther down.